packages/phone/src/Phone.stories.tsx

```diff
@@ -70,3 +74,20 @@ Default.args = {
   extColProps: { xs: { size: 3 } },
 };
 Default.storyName = 'default';
+
+export const hidden_FieldProps = ({}: FieldProps): JSX.Element => {
+  return <></>;
+};
+
+export const Props: Story = () => (
+  <>
+    <h4>Availity Props</h4>
+    <h5>Phone</h5>
+    <ArgsTable of={Phone} />
+
+    <h4>Availity Form Props</h4>
```

Comment:
Since this table doesn't list descriptions/defaults I think it's better to remove it since this is an availity component and can be found on this site. (although you can add a note above the Phone ArgsTable that any additional props passed to @availity/form field component.